content/attendee-run.md

```diff
@@ -11,3 +11,12 @@ Here are some links to attendee-run APIs:
   - 2022: [https://2022.schedule.emfcamp.dan-nixon.com/](https://2022.schedule.emfcamp.dan-nixon.com/)
 * [GCHQ.NET: Great Camp Hexpansion Quest API](https://gchq.net/api/docs/)
 * [MQTT schedule announcements](https://github.com/DanNixon/emfcamp-2024/blob/main/cloud-apps/schedule-mqtt-announcer/README.md)
+* Open EMF Camper
+  - [drinks](https://emf.harvinderatwal.com/beer?events=drink)
```

Comment:
I think you wanted ?event= rather than ?events= here (which gives an Internal Server Error response)